src/seth/libexec/seth/seth

```diff
@@ -153,6 +156,8 @@ while [[ $1 ]]; do
     -G|--gas)        shift; export ETH_GAS=$1;;
     -P|--gas-price)  shift;
       export ETH_GAS_PRICE=$(seth --to-wei ${1%gwei} ${1##*([0-9])});;
+    --prio-fee)      shift;
```

Comment:
is the `gas price` going to be used as `max fee` when the tip is set?

Reply:
yes

Reply:
```suggestion
    -T|--prio-fee)   shift;
```